package.json

```diff
@@ -14,7 +14,7 @@
     "screenshots": "lerna run screenshots --stream --concurrency 1",
     "start": "lerna run bundle && npm start --prefix ./packages/d3fc",
     "commitlint": "commitlint --from HEAD~1 to HEAD --verbose",
-    "examples": "cd examples && npx jest"
+    "examples": "cd examples && npx jest -i"
```

Comment:
Please can you use the long version of the run in band flag? Long flags makes it easier for the next person reading the script, short flags make it easier for you to type into a console 😄